### PR TITLE
Added python code in Connecting two Barns

### PR DESCRIPTION
**Pull request by 1a1242**
Python code was not available in connecting two barns so added it in usaco-1159.mdx

_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/usaco-1159.mdx [hunk 1]
```diff
@@ -144,6 +144,41 @@ public class ConnectingTwoBarns {
...
+		visit_neighbour(neighbour)
+
+for _ in range(int(input())):
+
+	N, M = map(int, input().split())
+	visited = [False] * (N)
+	roads = [[] for _ in range(N)]
+
+	for i in range(M):
+
```
Comment (envyaims) [inline solutions/usaco-1159.mdx:165]: what are these random spaces for?

**Review by envyaims — CHANGES_REQUESTED** [review]
r u sure this is correct?

**Comment by 1a1242** [comment]
> r u sure this is correct?

Yeah I checked with few random test cases and even the sample test cases it was working fine

File: solutions/usaco-1159.mdx (hunk 1 above)
Comment (1a1242) [reply solutions/usaco-1159.mdx:165]: Will remove it

Comment (1a1242) [reply solutions/usaco-1159.mdx]: I just wanted to make it more readable separating the loops and conditional statements.

**Comment by 1a1242** [comment]
> r u sure this is correct?

Sorry I understood the question wrong.
Will close this pull request now.

**Pull request closed without merging**